src/bpftrace.h

```diff
@@ -66,11 +67,10 @@ class WildcardException : public std::exception
 class BPFtrace
 {
 public:
-  BPFtrace(std::ostream& o = std::cout) : out_(o),ncpus_(get_possible_cpus().size()) { }
+  BPFtrace(std::unique_ptr<Output> o = std::make_unique<TextOutput>(std::cout)) : out_(move(o)),ncpus_(get_possible_cpus().size()) { }
```

Comment:
std::move seems more consistent with the rest of the codebase
```suggestion
  BPFtrace(std::unique_ptr<Output> o = std::make_unique<TextOutput>(std::cout)) : out_(std::move(o)),ncpus_(get_possible_cpus().size()) { }
```